core/domain/change_domain.py

```diff
@@ -22,21 +22,30 @@
 
 from core import feconf
 from core import utils
-from core.domain import state_domain
 
 from typing import Any, Dict, List, Mapping, Union, cast
 
-# Union type defined from allowed types that a Dict can contain for its values.
-AcceptableChangeDictTypes = Union[
-    str,
-    bool,
-    int,
-    List[str],
-    List[state_domain.AnswerGroupDict],
-    state_domain.RecordedVoiceoversDict,
-    # This Dict type is added to allow BaseChange to accept customization_args.
-    Dict[str, Dict[str, Any]]
-]
+MYPY = False
+if MYPY: # pragma: no cover
+    from core.domain import platform_parameter_domain
+    from core.domain import state_domain
+
+    # Union type defined from allowed types that a Dict can contain
+    # for its values.
+    AcceptableChangeDictTypes = Union[
+        str,
+        bool,
+        int,
+        float,
+        None,
+        List[str],
+        List[Dict[str, Any]],
+        Dict[str, Any],
+        state_domain.SubtitledHtmlDict,
+        List[state_domain.AnswerGroupDict],
+        List[platform_parameter_domain.PlatformParameterRuleDict],
+        state_domain.RecordedVoiceoversDict,
+    ]
```

Comment:
I did this change because importing `platform_parameter_domain` normally throws an import error. So to avoid the import error I imported `platform_parameter_domain` in `if MYPY:` clause.

`AcceptableChangeDictTypes` also shifted under `if MYPY:` clause because after shifting `platform_parameter_domain` to `if` clause backend test throws an error `platform_parameter_domain is not defined` ( which is expected because code written under `if MYPY:` clause does not execute at runtime. ).

Also, `AcceptableChangeDictTypes` is only defined for type checking so i think it's not a problem if we shift it under the `if` clause because from `AcceptableChangeDictTypes` it is not expected to execute at runtime.

Reply:
added